packages/quantic/force-app/main/default/lwc/quanticTab/e2e/fixture.ts

```diff
@@ -0,0 +1,61 @@
+import {TabObject} from './tabObject';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+
+const tabUrl = 's/quantic-tab';
```

Comment:
we can stick with pageUrl to stay consistent.  
```suggestion
const pageUrl = 's/quantic-tab';
```

Reply:
sure